static/app/views/organizationStats/index.tsx

```diff
@@ -324,6 +324,15 @@ export class OrganizationStats extends Component<OrganizationStatsProps> {
       if (opt.value === DATA_CATEGORY_INFO.metrics.plural) {
         return hasCustomMetrics(organization);
       }
+      if (
+        DATA_CATEGORY_INFO.profileDuration.plural === opt.value ||
+        DATA_CATEGORY_INFO.span.plural === opt.value
+      ) {
+        return organization.features.includes('spans-first-ui');
+      }
+      if (DATA_CATEGORY_INFO.transaction.plural === opt.value) {
+        return !organization.features.includes('spans-first-ui');
```

Comment:
asking to see if i can use this flag for this, otherwise i'll need to add another

Reply:
Good question. I added Performance team for review.